phpstan.neon.dist

```diff
@@ -0,0 +1,32 @@
+parameters:
+    level: 1
+    paths:
+        - ./
+    excludePaths:
+        - vendor/
+        # remove once https://github.com/phpstan/phpstan/issues/7955 is fixed
+        - fonts/
+
+    scanFiles:
+      - tcpdf_autoconfig.php
+
+    ignoreErrors:
+        - '~^Constant (K_PATH_URL|K_PATH_FONTS) not found\.$~'
+        - '~^Constructor of class TCPDF has an unused parameter \$diskcache\.$~'
+        - '~^Variable \$\w+ might not be defined\.$~'
+        - '~^Method TCPDF(_FILTERS)?::\w+\(\) should return .+ but return statement is missing\.$~'
+
+        # mcrypt does not support PHP 7.2 or later
+        -
+            message: '~^(Constant MCRYPT_RIJNDAEL_128 not found\.|Function mcrypt_get_cipher_name not found\.)$~'
+            path: tcpdf.php
+            count: 2
+        -
+            message: '~^(Constant (MCRYPT_ARCFOUR|MCRYPT_MODE_CBC|MCRYPT_MODE_STREAM|MCRYPT_RAND|MCRYPT_RIJNDAEL_128) not found\.|Function (mcrypt_create_iv|mcrypt_encrypt|mcrypt_get_iv_size) not found\.)$~'
+            path: include/tcpdf_static.php
+            count: 16
+
+        -
+            message: '~^(Call to static method create\(\) on an unknown class Symfony\\Component\\Finder\\Finder\.|Instantiated class Doctum\\(Doctum|RemoteRepository\\GitHubRemoteRepository) not found\.)$~'
+            path: scripts/doctum.php
```

Comment:
you could exclude this config file, it will be clearer

Reply:
I prefer to keep it for other possible static errors.